### DISCUSSION TITLE
How can I select the current scope or line with a keybind?

### DISCUSSION THREAD

**Discussion by ghost** [LazyVim/LazyVim#3330] [Q&A]
I really loved the feature in JetBrains and just about any other idea where if you do CTRL + C and CTRL + V it will copy and paste the current line below for each time you paste.

Can I get this in LazyVim?
Also I would love to be able to copy the active scope, so if I am within some brackets I can highlight and then yank the scope at different levels?

**Comment by dpetka2001** [comment]
`:h text-objects` and `mini.ai` are your friends.
